tests/System.Text.Primitives.Tests/Formatting/TimeTests.cs

```diff
@@ -51,6 +53,7 @@ public void RandomDateTimeTests(char format)
         static void TestDateTimeFormat(DateTime dt, char format, SymbolTable symbolTable)
         {
             var expected = dt.ToString(format.ToString(), CultureInfo.InvariantCulture);
+            if (format == 'r') expected = expected.ToLowerInvariant();
```

Comment:
@Tornhoof, this whole package is about doing formatting directly to byte buffers (i.e. bypassing string formatting).